### PR TITLE
Error handling for getGitHubApiResponse

### LINKED ISSUE #157: Error handling for getGitHubApiResponse
**Describe the bug**
When you don't have SSL installed, getGitHubApiResponse() returns false

**To Reproduce**
1. Composer start
2. No SSL certificate installed
3. Try http://localhost:8000/?user=DenverCoder1

**Expected behavior**
It should make a normal request to Git.

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - Version 20H2

**Additional context**
Add to documentation

### PR DESCRIPTION
**Pull request by komen205**
## Description

IF it's bool false, then throws an error. If has a specific string from an SSL error certificate, throws a different exception.
Fixes #157

### Type of change

- [X] New feature (added a non-breaking change which adds functionality)

## How Has This Been Tested?

- [ ] Tested locally with a valid username
- [ ] Tested locally with an invalid username
- [X] Ran tests with `composer test`
- [ ] Added or updated test cases to test new features

## Checklist:

- [X] I have checked to make sure no other [pull requests](https://github.com/DenverCoder1/github-readme-streak-stats/pulls?q=is%3Apr+sort%3Aupdated-desc+) are open for this issue
- [X] The code is properly formatted and is consistent with the existing code style
- [X] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

### CODE REVIEW

File: src/stats.php [hunk 1]
```diff
@@ -108,6 +108,16 @@ function getGitHubApiResponse(string $url): string
     curl_setopt($ch, CURLOPT_VERBOSE, false);
     curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, true);
     $response = curl_exec($ch);
+
+    //Handles curl errors
+    if($response==false){
+        if(str_contains(curl_error($ch),'unable to get local issuer certificate'))
+        {
+            throw new InvalidArgumentException("You don't have valid SSL Certificate installed or XAMPP.");
```
Comment (DenverCoder1) [inline src/stats.php:116]: Maybe this should display an error card (svg/png) like the other errors?

Comment (komen205) [reply src/stats.php:116]: I think if you throw, it will show the card. Atleast for me, shows me this.
image

Comment (DenverCoder1) [reply src/stats.php:116]: Ok, yeah that's right. It's in a try-catch that does it.

File: src/stats.php [hunk 2]
```diff
@@ -108,6 +108,16 @@ function getGitHubApiResponse(string $url): string
...
+        }
+        throw new InvalidArgumentException("Something is wrong with getGitHubApiResponse().");
+    }
```
Comment (DenverCoder1) [inline src/stats.php:119]: Formatting and triple equals for comparing with false literal

```suggestion
    // Handles curl errors
    if ($response === false) {
        if (str_contains(curl_error($ch), "Unable to get local issuer certificate.")) {
            throw new InvalidArgumentException("You don't have a valid SSL Certificate or XAMPP installed.");
        }
        throw new InvalidArgumentException("An error occurred in getGitHubApiResponse().");
    }
```

**Review by DenverCoder1 — APPROVED** [review]
Thanks!

**Pull request merged**